src/main/java/com/solutions/done/VolumeOfCylinder.java

```diff
@@ -0,0 +1,34 @@
+package com.solutions.done;
+
+import java.util.Scanner;
+
+public class VolumeOfCylinder {
+    static double $radius, $height;
+    float $pi;
+    float $voc;
+
+    public VolumeOfCylinder() {
+        $pi = 3.14F;
+    }
+    private void radius(double radius){
+        $radius = Math.pow(radius, 2);
+    }
+    private void volume(double $radius, double $height){
+        $voc = (float) ($pi*$radius*$height);
```

Comment:
you should call the getRadius() method in the calculation here. I.e. ($pi*getRadius()*$height)